src/app/pages/MarginTradePage/components/OpenPositionsTable/index.tsx

```diff
@@ -23,7 +25,7 @@ export function OpenPositionsTable({ perPage }: Props) {
   const { value, loading } = useGetActiveLoans(
     useAccount(),
     0,
-    1000,
+    100,
```

Comment:
`loanType` has changed?

Reply:
It's not type but max items to be returned (pagination), as there is no users with some much open positions it's better to keep this number low - less stress on server, faster to load.